ynr/apps/candidates/tests/test_models.py

```diff
@@ -104,3 +149,87 @@ def test_has_results_and_no_results(self):
         all_winners = set(has_results + has_winner_no_result)
         self.assertEqual(set(Ballot.objects.has_results()), all_winners)
         self.assertEqual(set(Ballot.objects.no_results()), set(no_results))
+
+
+class TestBallotQuerySetMethods(
+    TestCase, BallotsWithResultsMixin, SingleBallotStatesMixin
```

Comment:
Only a minor thing but now that we split up the classes, you don't need to inherit the `BallotsWithResultsMixin` here